lib/lastbill_client/request/requests/job_document_request.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module KazeClient
+
+  # @author chevre_a@modulotech.fr
+  # Retrieve a job document from a job.
+  # @see KazeClient::Request
+  # @see KazeClient::Utils::FinalRequest
+  # @see KazeClient::Utils::AuthentifiedRequest
+  # @since 0.1.0
+  class JobDocumentRequest < Utils::FinalRequest
+
+    include Utils::AuthentifiedRequest
+    include Utils::ListRequest
```

Comment:
Are you sure you need to include `Utils::ListRequest`? This request looks like it returns *one* document, not a list.